docs/scripts_md/get_dicom_files.md

```diff
@@ -22,16 +22,20 @@ Available options are:
            the scan type name is not used to determine which DICOM files to extract. This option
            must be used if no patient name patterns were specified via `-n` (see above).
 
-\-d       : extract the files in directory `<dir_argument>/get_dicom_files.pl.<UNIX_process_number>`
-           For example with `-d /data/tmp`, the DICOM files will be extracted in 
-           `/data/tmp/get_dicom_files.pl.67888` (assuming 67888 is the process number). 
-           By default, dir\_argument is set to the value of the environment variable `TMPDIR`. Note 
-           that the directory argument has to exist and be writeable, otherwise the script will issue
-           an error and abort.
+\-d       : extract the files in directory `<dir_argument>/get_dicom_files.pl.<random_string>`
```

Comment:
is it possible to have long options instead of just letters like all the other scripts in LORIS-MRI? (a.k.a. -profile, -pscid -directory....)

Reply:
For compatibility, sure. I'll correct.

Reply:
Thanks!